lib/init.g

```diff
@@ -627,10 +627,8 @@ ReadOrComplete( "lib/read4.g" );
 #  around function definitions!
 ReadLib( "helpview.gi"  );
 
-#T  1996/09/01 M.Schönert this helps performance
-IMPLICATIONS:=IMPLICATIONS{[Length(IMPLICATIONS),Length(IMPLICATIONS)-1..1]};
 # allow type determination of IMPLICATIONS without using it
-TypeObj(IMPLICATIONS[1]);
+TypeObj(IMPLICATIONS_COMPOSED[1]);
 #T shouldn't this better be at the end of reading the library?
 #T and what about implications installed in packages?
 #T (put later installations to the front?)
```

Comment:
These three `#T` comment lines reference the reversing of `IMPLICATIONS`, not the `TypeObj` call (I verified it by looking at the commit history: Thomas Breuer added the first one 2005-04-15, the second 2010-05-19, the third 2010-07-28;  while Frank squeezed the two type determination line in 2005-07-18)

Thus, if the reversal is removed, these three comment lines should be removed, too.